config/clusterctl-templates/cluster-template.yaml

```diff
@@ -78,6 +84,7 @@ spec:
         advertiseAddress: 127.0.0.1
         bindPort: 443
       nodeRegistration:
+        taints: []
```

Comment:
These taints should be moved into a custom template sitting within the tests folder.
With the empty taints, Control Plane nodes will be schedulable by default leading to addtionally usable nodes.